### PR TITLE
Update RunQueryActionButton.jsx

### PR DESCRIPTION
**Pull request by nabinkhadka**
Simple tooltip text enhancement

### CODE REVIEW

File: superset/assets/src/SqlLab/components/RunQueryActionButton.jsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default function RunQueryActionButton(props) {
         {...commonBtnProps}
         onClick={() => props.runQuery(true)}
         key="run-async-btn"
-        tooltip={t('Run query asynchronously')}
+        tooltip={t('Run query asynchronously(Ctrl + ↵)')}
```
Comment (etr2460) [inline superset/assets/src/SqlLab/components/RunQueryActionButton.jsx]: nit: add a space before the open paren

**Review by etr2460 — COMMENTED** [review]
you'll also want to add this to the tooltip on the non async query button too.

Although I'd question if this is necessary since the keyboard shortcuts are already shown in the UI
image

File: superset/assets/src/SqlLab/components/RunQueryActionButton.jsx (hunk 1 above)
Comment (nabinkhadka) [reply superset/assets/src/SqlLab/components/RunQueryActionButton.jsx]: @etr2460 Thanks. Done.

**Comment by nabinkhadka** [comment]
> you'll also want to add this to the tooltip on the non async query button too.
> 
> Although I'd question if this is necessary since the keyboard shortcuts are already shown in the UI
> image

As a user, I like to know the shortcut when I hover it. I think we like to see it right there. Please let me know. Thanks

**Comment by etr2460** [comment]
can you change your pr to merge into master?

**Pull request closed without merging**